src/ontology/uberon-edit.obo

```diff
@@ -224513,6 +224513,18 @@ relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-
 relationship: located_in UBERON:8600025 ! intervillous space
 property_value: dcterms-date "2024-04-15T10:05:35Z" xsd:dateTime
 
+[Term]
+id: UBERON:8900000
+name: sensory corpuscle
+def: "A mechanosensory complex in the terminal end of some nerves that forms a corpuscle. It consists of one or more neuronal terminations and associated non-neuronal cells." [PMID:35356056]
+synonym: "COEC" EXACT [PMID:35356056]
+synonym: "cutaneous end-organ complex" EXACT [PMID:35356056]
+xref: http://www.wikidata.org/entity/Q126363978
+is_a: UBERON:0005162 ! multi cell part structure
```

Comment:
The definition of `multi cell part structure` is:
> A structure consisting of multiple cell components but which is not itself a cell and does not have (complete) cells as a part.

In sensory corpuscles there are some complete cells. I'm not aware if there is a term that comprises parts of cells and complete cells, but I will have a look.

Reply:
> The definition of `multi cell part structure` is:
> 
> > A structure consisting of multiple cell components but which is not itself a cell and does not have (complete) cells as a part.
> 
> In sensory corpuscles there are some complete cells. I'm not aware if there is a term that comprises parts of cells and complete cells, but I will have a look.

Oh, curious. I had not seen the definition, I interpreted quite the oposite from the label!! 

So apparently the capsule of the Pacinian corpuscle is being excluded altogether from the Pacinian corpuscle itself.

I'll copy that in the issue.